docs/csa/threat-detection-and-investigation.md

```diff
@@ -0,0 +1,310 @@
+1. Click on the three vertical dots in the upper right corner of the **Top 10 User Activity** panel.
+1. Select the **Edit** option so you can modify your query. 
+1. We want to filter by event_type in our panel, but allow the dashboard viewer options to pick the event type. Add a new line to the query by pressing Shift + Enter and type or copy the following code:
+   ```
+   | where event_type matches "{{event_type}}"
+   | where actor matches "{{actor}}"
+   ```
+ Your query should now look something like this:<br/><img src={useBaseUrl('img/csa/new-query.png')} alt="Revised query" style={{border: '1px solid black'}} width="400"/>
+ 1. Click the **Update Dashboard** button. <br/>You can test these changes by selecting different **event_types** and **actors** from the template fields in the dashboard bar. As you select different values you will see the dashboard panel change automatically. Select the ** * ** option as a variable to see all options.  When you are done testing, ensure both **event_type** and **actor** template variables are set to * before continuing. 
+
+### Step 3: Monitor geolocation of console logins
+
+The [lookup](/docs/search/search-query-language/search-operators/lookup-classic/) operator maps data to other meaningful data stored in Sumo Logic.
+
+To see where your users are logging in from across the world, you can use [geo lookups](/docs/search/search-query-language/search-operators/geo-lookup-map/). You can create maps with geo lookups, which can help you visualize and quickly identify where users are logging in from. You can then use this information to ensure all logins are from expected locations, and detect potential login threats.
+
+For example, if you expect all your users to be located in Europe, but you are getting a large number of logins from South America, this might be worth investigating. There could be potential hackers in the unexpected location, or it could be a sign that an advertising campaign was accidentally deployed in the wrong region. In this step, we'll create a map that monitors where users are logging in from.
+
+As another example, let's say one of your employees logged in from both Canada and Australia, on the same day only two hours apart. You know this employee lives in Toronto and travels frequently. So, the Canadian login is expected. Normally, the Australian login would be expected too, since this employee travels so often. However, since it occurred only two hours after a Toronto login, you conclude it's  malicious activity. This type of suspicious activity is called a landspeed violation. We'll show you how to query for them.
```

Comment:
```suggestion
As another example, let's say one of your employees logged in from both Canada and Australia, on the same day only two hours apart. You know this employee lives in Toronto and travels frequently. So, the Canadian login is expected. Normally, the Australian login would be expected too, since this employee travels so often. However, since it occurred only two hours after a Toronto login, you conclude it's malicious activity. This type of suspicious activity is called a landspeed violation. We'll show you how to query for them.
```